configure.ac

```diff
@@ -7,7 +7,7 @@
 #	Process this file with autoconf to produce a configure script.
 
 AC_PREREQ([2.64])
-AC_INIT([universal-ctags],[6.0.0])
+AC_INIT([universal-ctags],[6.0.1])
```

Comment:
Don't edit this.

Reply:
I apologize - I attempted to follow the online recommendations, but makes more sense for that to be only for the integral editors/authors.